theano/sandbox/cuda/basic_ops.py

```diff
@@ -3829,9 +3823,9 @@ def do_constant_folding(self, node):
                 # If the output is a constant, it will have to be deepcopied
                 # each time the function is called.  So we do not fold.
                 return False
-            elif (  # The following ops work inplace of their input id 0.
-                  client[1] == 0 and
-                  isinstance(client[0].op, (
```

Comment:
What was the pep8 problem here? The solution with over-indented return is a bit weird. Could you rather over-indent the elif statement?